Detectors/TRD/workflow/src/TRDDigitizerSpec.cxx

```diff
@@ -28,6 +28,7 @@
 #include "DetectorsBase/BaseDPLDigitizer.h"
 #include "TRDBase/Calibrations.h"
 #include "DataFormatsTRD/TriggerRecord.h"
+#include "DataFormatsTRD/HitType.h"
```

Comment:
Same as above, please check the order of the headers.

Reply:
Done.
Given that the style follows what was originally there with regards to TriggerRecord and there are TRDsimulation includes between TRDbase includes, root includes amongst O2 includes, which style guide are we following?

Reply:
We should follow Google Style: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

> For example, the includes in google-awesome-project/src/foo/internal/fooserver.cc might look like this:
> ```
> #include "foo/server/fooserver.h"
> 
> #include <sys/types.h>
> #include <unistd.h>
> 
> #include <string>
> #include <vector>
> 
> #include "base/basictypes.h"
> #include "base/commandlineflags.h"
> #include "foo/server/bar.h"
> ```

> Exception:
> 
> Sometimes, system-specific code needs conditional includes. Such code can put conditional includes after other includes. Of course, keep your system-specific code small and localized. Example:
> ```
> #include "foo/public/fooserver.h"
> 
> #include "base/port.h"  // For LANG_CXX11.
> 
> #ifdef LANG_CXX11
> #include <initializer_list>
> #endif  // LANG_CXX11
> ```

Reply:
Ja of course order can matter